cost-analyzer/templates/_helpers.tpl

```diff
@@ -212,9 +212,9 @@ app: query-service
 {{ include "kubecost.federator.chartLabels" . }}
 app: federator
 {{- end -}}
-{{- define "waterfowl.commonLabels" -}}
-{{ include "kubecost.waterfowl.chartLabels" . }}
-app: waterfowl
+{{- define "aggregator.commonLabels" -}}
+{{ include "kubecost.aggregator.chartLabels" . }}
+app: kubecost-query-handler
```

Comment:
👀

Reply:
one rename too far... thank you for catching this 🙈